docs/src/main/sphinx/sql/create-catalog.md

```diff
@@ -0,0 +1,76 @@
+# CREATE CATALOG
+
+## Synopsis
+
+```text
+CREATE CATALOG
+catalog_name
+USING connector_name
+[ WITH ( property_name = expression [, ...] ) ]
+```
+
+## Description
+
+Create a new catalog using the specified connector. The catalog is ready to be
+used as soon as this query completes.
+
+The optional `WITH` clause is used to set properties on the newly created
+catalog. Property names can be double quoted, which is required if they contain
+special characters, like `-`. Refer to [connectors documentation](/connector)
+to learn about all available properties. All property values must be varchars
+(single quoted), including numbers and boolean values.
+
+The query will fail if:
+* A required property is missing.
+* An invalid property is set, for example there is a typo in the property name,
+  or a property name from a different connector was used.
+* The value of the property is invalid, for example a numeric value is out of
+  range, or a string value doesn't match the required pattern.
+* The value references an environmental variable that is not set on the
+  coordinator node.
+
+:::{warning}
+The complete `CREATE CATALOG` query is logged, and visible in the {doc}`Web UI
+</admin/web-interface>`. This includes any sensitive properties, like passwords
+and other credentials. See {doc}`secrets </security/secrets>`.
+:::
+
+:::{note}
+This command requires [catalog management type](/admin/properties-catalog) to
+be set to `dynamic`.
+:::
+
+## Examples
+
+Create a new catalog called `tpch` using the [](/connector/tpch):
+
+```sql
+CREATE CATALOG tpch USING tpch;
+```
+
+Create a new catalog called `brain` using the [](/connector/memory):
+
+```sql
+CREATE CATALOG brain USING memory
+WITH ("memory.max-data-per-node" = '128MB');
+```
+
```

Comment:
Note the required use of " for quoting the property name, since the dash characters in the property name are invalid in SQL. The value of 128MB must be quoted with standard '. sin and the required ' to